### PR TITLE
Add legacy_sql_tranlsation_helper

### PR DESCRIPTION
**Pull request by celiajmd**

### CODE REVIEW

File: tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        table_name, dataset_id, project_id = self.parse_table_name(table)
+
+        exist_col_type = {}
+        client = bigquery.Client()
+        query = """select
+        column_name, data_type
+        from {}.{}.INFORMATION_SCHEMA.COLUMNS
+        where table_name = {}""".format(project_id,dataset_id,table_name )
+
+        query_i_s = client.query(query)
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py:72]: Don't run a query for this, simply make an API call to the tables.get endpoint

File: tools/legacy_sql_tranlsation_helper/readSQLToString.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import os
+
+class readSQLToString:
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/readSQLToString.py:3]: No need to make this a class, just make it a function

File: tools/legacy_sql_tranlsation_helper/LICENSE [hunk 3]
```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/LICENSE:1]: you can remove this license

File: tools/legacy_sql_tranlsation_helper/patternScan.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
+import re
+from readSQLToString import *
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/patternScan.py:2]:
```suggestion
import readSQLToString
```

File: tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py [hunk 5]
```diff
@@ -0,0 +1,141 @@
+from google.cloud import bigquery
+from readSQLToString import *
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py:2]:
```suggestion
import readSQLToString
```

File: tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+import re
+
+legacy_sql = """Replace your SQL here. """
+file_path = "sample_sql/sample1.sql"
+
+if file_path != "" and legacy_sql == "":
+    legacy_sql = read_file_path(file_path)
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/translatorCommaJoin.py:9]:
```suggestion
    legacy_sql = readSQLToString.read_file_path(file_path)
```

File: tools/legacy_sql_tranlsation_helper/readSQLToString.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+#check if file is present
+def read_file_path(file_path):
+    if os.path.isfile(file_path):
+        #open text file in read mode
+        text_file = open(file_path, "r")
+        #read whole file to a string
+        data = text_file.read()
+        #close file
+        text_file.close()
+        return data
```
Comment (danieldeleo) [inline tools/legacy_sql_tranlsation_helper/readSQLToString.py:12]:
```suggestion
        #open text file in read mode
        with open(file_path, "r") as text_file:
            #read whole file to a string
            return  text_file.read()
```

**Review by rajc242 — APPROVED** [review]

**Pull request merged**